### ISSUE TITLE
NANs in high learning rates

### ISSUE THREAD

**Issue by riley-mld** [EmbarkStudios/emote#20]
Initially this problem was more severe and even in low learning rates after a couple of hours of training we would get NANs, but after applying orthogonal weight initialisation and setting bias to a constant 0, we no longer get NANs with lower learning rates, but we still get them very early into the training when the learning rate is set higher than about 3e-3.
Exploding gradients seems like a likely candidate but this hasn't be tested and we clip gradients so this could be another issue potentially. As @jorgdv suggested SquashStretchTransform on actions is a good place to start looking into.
